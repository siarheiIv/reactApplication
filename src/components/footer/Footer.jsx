import React from 'react';
import Link from 'next/link';

const Footer = () => (
        <footer className="footer">
            <div className="wrapper">
                <Link href="/"><a className="logo">netflixroulette</a></Link>
            </div>
        </footer>);

export default Footer;
