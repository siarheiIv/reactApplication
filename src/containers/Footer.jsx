import React from 'react';

const Header = () => {
    return(
            <footer className="footer">
                <div className="wrapper">
                    <a href="#" className="logo">netflixroulette</a>
                </div>
            </footer>
    )
}

export default Header;