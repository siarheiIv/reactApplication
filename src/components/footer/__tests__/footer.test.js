import React from 'react';
import { render } from '@testing-library/react';
import Footer from '../index';

describe('Footer', () => {
    it('should render Footer', () => {
        const { asFragment } = render(<Footer />);
        expect(asFragment(<Footer />)).toMatchSnapshot();
    })
})