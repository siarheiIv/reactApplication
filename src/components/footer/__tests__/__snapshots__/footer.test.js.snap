// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render Footer 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <div
      class="wrapper"
    >
      <a
        class="logo"
        href="#"
      >
        netflixroulette
      </a>
    </div>
  </footer>
</DocumentFragment>
`;
